src/apps/bm_devkit/bm_loadcell/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ set(LIB_FILES
     ${SRC_DIR}/lib/common/uptime.c
     ${SRC_DIR}/lib/common/util.c
     ${SRC_DIR}/lib/common/watchdog.c
+    ${SRC_DIR}/lib/common/ll.c
```

Comment:
@towynlin  Just curious, and maybe we can have this discussion at another time, but why do we not build the lib/bsp/third_party sources as libraries separately and then link them in? I feel like a lot of redundancy could be cut down here to ensure that new source files do not have to be added to to each app, but that's an assumption and I do not have the full background of the why!

Reply:
No particular reason — has occurred to me too — could totally do that in a separate PR.